docs/contracts/visibility-and-getters.rst

```diff
@@ -186,39 +194,28 @@ to write a function, for example:
         }
     }
 
-Now you can use ``getArray()`` to retrieve the entire array, instead of
-``myArray(i)``, which returns a single element per call.
-
-The next example is more complex:
-
-.. code-block:: solidity
-
-    // SPDX-License-Identifier: GPL-3.0
-    pragma solidity >=0.4.0 <0.9.0;
+    contract StructExample {
+        // Struct Definition with single member
+        struct SimpleStruct {
+            int[2] arr;
+        }
 
-    contract Complex {
-        struct Data {
+        // Struct Definition with multiple members
+        struct ComplexStruct {
             uint a;
-            bytes3 b;
-            mapping(uint => uint) map;
-            uint[3] c;
-            uint[] d;
-            bytes e;
+            bool b;
+            int[2] arr;
         }
-        mapping(uint => mapping(bool => Data[])) public data;
-    }
 
-It generates a function of the following form. The mapping and arrays (with the
-exception of byte arrays) in the struct are omitted because there is no good way
-to select individual struct members or provide a key for the mapping:
```

Comment:
I think this should stay.

Reply:
I have merged it with the `ComplexStruct`